### PR TITLE
Try to fix Pill widget when Google Now page disabled (WIP)

### PR DESCRIPTION
**Pull request by divadsn**
Signed-off-by: David Sn <divad.nnamtdeis@gmail.com>

To summarize, here is what I'm doing right now:
- [x] Google Pill widget should work regardless of Lawnfeed enabled/installed (search on tap)
- [x] mLauncher should be only connected **if** the user has Google Now enabled
- [ ] ~~Fix animation being not shown on Play Store version~~
- [x] Make parts of the widget toggleable, like the pill, edge or weather/date (not wide bar)

### CODE REVIEW

**Comment by divadsn** [comment]
There is still work left to do, there is a logic problem where LawnfeedClient is assumed to do the Google Now page and Pill widget, while the other implementation is only for the Google Now page.

In the end we need to make it possible to use the pill widget, while having the Google Now page properly disabled for working scroll.

Maybe removing the Google Now page check in LawnfeedClient would fix it better.

**Comment by divadsn** [comment]
To summarize, here is what I'm doing right now:
- Google Pill widget should work regardless of Lawnfeed enabled/installed (search on tap)
- mLauncher should be only initiated **if** the user has Google Now enabled
- If no Google app installed, the search should open google.com in default browser
- Also, open weather details in browser if Google not installed (what about OWM?)
- Disable Awareness API if no Google Play services

**Comment by nyancrimew** [comment]
@divadsn opening weather in the browser could simply be done by opening https://www.google.ch/search?q=weather in the browser. This would be the closest to opening the weather in the Google app we can do in the browser

**Comment by divadsn** [comment]
But do we store somewhere the closest OWM location from the API to open that in the browser too?

**Comment by nyancrimew** [comment]
@divadsn OWM currently only works with the custom location entered in the settings, so yes, we do store that 😉

**Comment by divadsn** [comment]
Added customization options to the list, weather related stuff in another PR.

File: app/src/main/java/ch/deletescape/lawnchair/pixelify/BaseQsbView.java [hunk 1]
```diff
@@ -127,13 +131,22 @@ private void initializeQsbConnector() {
 
     @Override
     public void onClick(View view) {
-        if (view.getId() == R.id.mic_icon) {
-            startQsbActivity(VOICE_ASSIST);
-        } else if (BuildConfig.ENABLE_LAWNFEED && PackageManagerHelper.isAppEnabled(getContext().getPackageManager(), LawnfeedClient.PROXY_PACKAGE, 0)) {
-            ((LawnfeedClient) mLauncher.getClient()).onQsbClick(bm("com.google.nexuslauncher.FAST_TEXT_SEARCH"), new Receiver(this));
-        } else {
-            getContext().sendOrderedBroadcast(bm("com.google.nexuslauncher.FAST_TEXT_SEARCH"), null, new C0287l(this), null, 0, null, null);
+        switch (view.getId()) {
```
[the remark this answers is not in the document]
Comment (divadsn) [reply app/src/main/java/ch/deletescape/lawnchair/pixelify/BaseQsbView.java]: BOI

**Pull request merged**